corehq/apps/analytics/tasks.py

```diff
@@ -368,7 +369,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user.get('email')
-        if not email:
+        if not email or not validate_email(email, verify=True):
```

Comment:
We may also want to make this configurable for the sake of NIC (which may not have good internet access to do the validation).